lib/ui/myplot/my_plot_child_item.dart

```diff
@@ -1,49 +1,49 @@
 import 'package:farmsmart_flutter/utils/styles.dart';
 import 'package:flutter/material.dart';
 
 class MyPlotListItem {
   Widget buildListItem(CropEntity cropsData, goToDetail) {
-    return GestureDetector(
-        onTap: () {
-          goToDetail(cropsData);
-        },
-        child: Padding(
-            key: ValueKey(cropsData.name ?? "No Title"),
-            padding: Margins.boxPadding(),
-            child: Container(
-                decoration: BoxDecoration(
-                  color: Colors.white,
-                  borderRadius: BorderRadius.circular(8.0),
-                  boxShadow: BoxShadows.plotListItemShadow()
+    return Padding(
+        key: ValueKey(cropsData.name ?? "No Title"),
+        padding: Margins.boxPadding(),
+        child: Container(
+            decoration: BoxDecoration(
+                color: Color(white),
+                borderRadius: BorderRadius.circular(8.0),
+                boxShadow: BoxShadows.plotListItemShadow()),
+            child: Column(children: <Widget>[
+              ClipRRect(
+                borderRadius: BorderRadius.only(
+                  topLeft: Radius.circular(8.0),
+                  topRight: Radius.circular(8.0),
                 ),
-                child: Column(
+                child: FadeInImage.assetNetwork(
+                    image: cropsData.imageUrl,
+                    height: listImageHeight,
+                    width: listImageWidth,
+                    placeholder: Assets.IMAGE_PLACE_HOLDER,
+                    fit: BoxFit.fitWidth),
+              ),
+              Padding(
+                  padding: Margins.boxBigPadding(),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
-                  ClipRRect(
-                    borderRadius: BorderRadius.only(
-                      topLeft: Radius.circular(8.0),
-                      topRight: Radius.circular(8.0),),
-//                      child: FadeInImage.assetNetwork(image: cropsData.imagePath, placeholder: 'assets/raw/crop_image_mock.png', fit: BoxFit.cover),
-                      child: Image.asset('assets/raw/crop_image_mock.png', fit: BoxFit.cover),
-                  ),
-                  Padding(
-                      padding: Margins.boxBigPadding(),
-                      child: Column(
-                        crossAxisAlignment: CrossAxisAlignment.start,
-                        children: <Widget>[
-                          Text(cropsData.name ?? "No Title", style: Styles.titleTextStyle()),
-                          Margins.generalListSmallMargin(),
-                          Text(Strings.myPlotCurrentStagesMOCK),
-                          Margins.generalListBigMargin(),
-                          MyPlotItemFooter().build()
-                        ],
-                      )),
-                ]))));
+                      Text(cropsData.name ?? "No Title",
```

Comment:
Can 'No Title' be defined as a constant somewhere in preparation of localisation?

Reply:
Sure!